creator_project/packages/creator-luacpp-support/reader/CreatorReader.cpp

```diff
@@ -377,21 +378,24 @@ cocos2d::Node* CreatorReader::createTree(const buffers::NodeTree* tree) const
 
     // recursively add its children
     const auto& children = tree->children();
-    for(const auto& childBuffer: *children) {
+    for(const auto& childBuffer: *children)
+    {
         cocos2d::Node* child = createTree(childBuffer);
-        if (child && node)  {
-            if (!treat_child_as_label) {
-                // every node should do this
-                node->addChild(child);
-                adjustPosition(child);
-            } else {
-                // except if for Buttons
+        if (child && node)
+        {
+            // should adjust child's position except Button's label
+            if (parsing_button && dynamic_cast<cocos2d::Label*>(child) != nullptr)
```

Comment:
This means that you're not able to add any child Labels of Buttons, without it being set as the title label. What if it was changed to also look for the name of the child, ie 'Label' the default value in the editor?

I might be using Creator wrong but it seems like you should be able to add any node to any child, and this gets in the way of that. If Layout gets supported, then that could get around this, but it seems like we could tighten this up one way or the other